test-it/ViaTests/StringFormatDefaultFormat.via

```diff
@@ -0,0 +1,69 @@
+// Autogenerated Vireo assembly file
+// Generated on 08/09/2019 13:59:40
```

Comment:
Please consider adding a test case where %s is being passed in the "format string" for the corresponding boolean arguments. As this was also incorrect before with respect to casing, it is worth capturing in a test.

Reply:
Done